CMakeLists.txt

```diff
@@ -37,8 +37,7 @@ set(CMAKE_CXX_STANDARD 14)
 set(DLAF_WITH_CUDA OFF CACHE BOOL "Enable CUDA support")
 if (DLAF_WITH_CUDA)
   enable_language(CUDA)
-
-  DLAF_addCudaTargets()
+  find_package(CUBLAS REQUIRED)
```

Comment:
Since cuda runtime and cublas are orthogonal libraries (there's no dependencies among them) I think that
it is misleading that `find_package(CUBLAS REQUIRED)` finds both.
I would change the name to something like `CUDALIBS`, such that other CUDA libraries can be added as well in case of need.

Reply:
The Find package finds `cublas` and it's dependencies, in that case `cudart`. Since we only need `cublas` for now, I don't see the point in exposing a second target or creating another Find package.

Reply:
In my knowledge it is not a dependency (cublas only depends on the driver). We need cuda runtime to manage the devices, streams, events, ... . Moreover we use it for allocations and copies (cublas has its own methods but the methods in cuda runtime are more flexible).
I agree to avoid to have a find module fora each cuda library we use, this is the reason I proposed to rename it to CUDALIBS which currently will include cuda runtime and cublas, but can be extended to other cuda libraries as needed.